### ISSUE TITLE
Deprecate IAmAXamarinMVP

### ISSUE THREAD

**Issue by Cheesebaron** [planetxamarin/planetxamarin#535]
It has been a while since all Xamarin MVP's were grandfathered over to Microsoft MVP status.

We should deprecate the IAmAXamarinMVP interface.

**Comment by Cheesebaron** [comment]
Yep. When that is done we can remove the Interface itself.

We should maybe also think about adding some kind of mechanism to verify MVP status. Probably a nice little task for integration tests.

**Comment by jfversluis** [comment]
Yeah, I was thinking about that the other day. Maybe include the MVP ID and simply check if https://mvp.microsoft.com/en-us/PublicProfile/id (e.g. https://mvp.microsoft.com/en-us/PublicProfile/5002166) yields an HTTP 200?
